src/qt/forms/optionsdialog.ui

```diff
@@ -104,6 +104,9 @@
         <layout class="QHBoxLayout" name="horizontalLayout_2_Main">
          <item>
           <widget class="QLabel" name="databaseCacheLabel">
+           <property name="toolTip">
+            <string extracomment="Tooltip text for Options window setting that sets the size of the database cache. Explains the corresponding effects of increasing/decreasing this value.">Limit the database cache size to a certain value. A larger cache size can contribute to faster sync times. Lowering the cache size will reduce memory usage.</string>
```

Comment:
Could this tooltip be aligned with https://github.com/bitcoin-core/gui/blob/ee1db7b6dc4e8a3458a7cfb5ead445c56fb75c51/src/init.cpp#L388 and mention "unused mempool memory"?

Reply:
Maybe something like this.
```suggestion
            <string extracomment="Tooltip text for Options window setting that sets the size of the database cache. Explains the corresponding effects of increasing/decreasing this value.">Maximum database cache size. A larger cache size can contribute to faster sync. Lowering the cache size will reduce memory usage. Unused mempool memory is shared for this cache.</string>
```

A sync can be faster, a sync time can be lower/higher, so maybe `s/faster sync time/faster sync/`

(Aside, ISTM the "Unused mempool memory is shared for this cache" text in `src/init.cpp` is not very clear, or at least not as precise as it could be.)

Reply:
In `doc/reduce-memory.md`, this may be worth appending:

"faster synce, after which the benefit is less pronounced for most use cases, unless fast validation of blocks is important, such as for mining."

Reply:
Thanks for the excellent review @jonatack, I've adopted your suggestions in 9bd168b